test/test_background_flux_divergence.jl

```diff
@@ -0,0 +1,78 @@
+using Oceananigans
+using Test
+using CUDA:has_cuda
+using NCDatasets
```

Comment:
```suggestion
include("depedencies_for_runtests.jl")
```

this is included in all test files which allows them to be run stand-alone.